### PR TITLE
WIP: Content edits

### PR DESCRIPTION
**Pull request by hlucey**
This PR is a WIP, please keep it open. Thanks!

### CODE REVIEW

**Comment by hlucey** [comment]
Readme Q: 

1. Line 135: What does this mean? 
> The name to be used on all Watson resources as prefix
 
2. Mentions of Terraform IBM-Cloud/ibm provider? Can this be rewritten to Terraform IBM Cloud and IBM providers?
 
3. Are there any other prefix limitations (For example, prefix must begin and end with a letter and contain only letters, numbers, and - characters.)

File: variables.tf [hunk 1]
```diff
@@ -69,7 +69,7 @@ variable "watson_studio_plan" {
   type        = string
   validation {
     condition     = contains(["free-v1", "professional-v1"], var.watson_studio_plan)
-    error_message = "The plan to use must be one of: free-v1 or professional-v1"
+    error_message = "The resource plan you use must be free-v1 or professional-v1"
```
Comment (andreainnocenti) [inline variables.tf:72]:
```suggestion
    error_message = "The resource plan you use must be free-v1 or professional-v1."
```

File: variables.tf [hunk 2]
```diff
@@ -33,13 +33,13 @@ variable "resource_group_name" {
 }
 
 variable "resource_prefix" {
-  description = "Name to be used on all Watson resource as prefix"
+  description = "The name to be used on all Watson resource as prefix"
```
Comment (andreainnocenti) [inline variables.tf:36]:
```suggestion
  description = "The name to be used on all Watson resource as prefix."
```

File: README.md [hunk 3]
```diff
@@ -137,20 +125,20 @@ statement instead the previous block.
...
+| <a name="input_watson_assistant_plan"></a> [watson\_assistant\_plan](#input\_watson\_assistant\_plan) | The resource plan used to provision the watsonx Assistance instance. | `string` | `"do not install"` | no |
+| <a name="input_watson_discovery_plan"></a> [watson\_discovery\_plan](#input\_watson\_discovery\_plan) | The resource plan used to provision the Watson Discovery instance. | `string` | `"do not install"` | no |
+| <a name="input_watson_governance_plan"></a> [watson\_governance\_plan](#input\_watson\_governance\_plan) | The resource plan used to provision the watsonx Governance instance. | `string` | `"do not install"` | no |
+| <a name="input_watson_machine_learning_plan"></a> [watson\_machine\_learning\_plan](#input\_watson\_machine\_learning\_plan) | The resource plan used to provision the Watson Machine Learning instance. | `string` | `"v2-standard"` | no |
+| <a name="input_watson_studio_plan"></a> [watson\_studio\_plan](#input\_watson\_studio\_plan) | The resource plan used to provision the Watson Studio instance. | `string` | `"professional-v1"` | no |
```
Comment (andreainnocenti) [inline README.md:141]: This documentation is automatically generated according to the `variables.tf` file. We should not modify anything manually here.

File: README.md [hunk 4]
```diff
@@ -45,23 +43,22 @@ https://terraform-ibm-modules.github.io/documentation/#/implementation-guideline
 The solution supports the following:
 
 - Creating a new resource group, or taking an existing one.
-- Provisioning the services
-  - `Watson Machine Learning`
-  - `Watson Studio`
-  - `Cloud Object Storage`.
-- Configuring the `IBM watsonx profile` and creating a *starter* `IBM watsonx project`
-  for an IBM Cloud user, who becomes the `admin` of the `IBM watsonx project` (`IBM watsonx admin` in the following).
+- Provisioning the follwoing services:
```
Comment (andreainnocenti) [inline README.md:46]:
```suggestion
- Provisioning the following services:
```

**Review by andreainnocenti — CHANGES_REQUESTED** [review]

**Review by mkrudele — CHANGES_REQUESTED** [review]
- Consider that there are sections of the readme that are automatically generated based on the variables.tf and outputs.tf, so you don’t need to update the related sections (doesn’t hurt, the sections will be simply overridden).  
- I saw you used entities in the README like `{{site.data.keyword.Bluemix_notm}}`. They are not rendered in the git repo, so perhaps we should not use in the README!? Not sure if they will be rendered in the conole catalog pages.  
- Would you mind to also edit the outputs.tf file?  
- Would you mind to also edit the nested modules `configure_project` and `configure_user` (same, variables.tf and outputs.tf)?  

Thanks, content looks much better now!!

**Review by mkrudele — APPROVED** [review]

**Review by mkrudele — DISMISSED** [review]

**Review by mkrudele — DISMISSED** [review]

**Review by mkrudele — APPROVED** [review]

**Review by andreainnocenti — APPROVED** [review]

**Pull request merged**